notifications.bs

```diff
@@ -802,92 +801,109 @@ attribute's getter must return the <a>maximum number of actions</a> supported.
-<a>notification</a>'s <a for=notification>badge URL</a>, <a lt="url serializer">serialized</a>, and
-the empty string if there is no <a>notification</a>'s <a for=notification>badge URL</a> otherwise.
+<ol>
+ <li><p>If there is no <a>this</a>'s <a>notification</a>'s <a for=notification>icon URL</a>, then
+ return the empty string.
 
-<p>The <dfn attribute for=Notification><code>vibrate</code></dfn> attribute's getter must return the
-<a>notification</a>'s <a for=notification>vibration pattern</a>, if any, and the empty list
-otherwise.
+ <li><p>Return <a>this</a>'s <a>notification</a>'s <a for=notification>icon URL</a>,
+ <a lt="URL serializer">serialized</a>.
+</ol>
 
-<p>The <dfn attribute for=Notification><code>timestamp</code></dfn> attribute's getter must return
-the <a>notification</a>'s <a for=notification>timestamp</a>.
+<p>The <dfn attribute for=Notification><code>badge</code></dfn> getter steps are:
 
-<p>The <dfn attribute for=Notification><code>renotify</code></dfn> attribute's getter must return
-the <a>notification</a>'s <a for=notification>renotify preference flag</a>.
+<ol>
+ <li><p>If there is no <a>this</a>'s <a>notification</a>'s <a for=notification>badge URL</a>, then
+ return the empty string.
 
-<p>The <dfn attribute for=Notification><code>silent</code></dfn> attribute's getter must return the
-<a>notification</a>'s <a for=notification>silent preference flag</a>.
+ <li><p>Return <a>this</a>'s <a>notification</a>'s <a for=notification>badge URL</a>,
+ <a lt="URL serializer">serialized</a>.
+</ol>
 
-<p>The <dfn attribute for=Notification><code>requireInteraction</code></dfn> attribute's getter must
-return the <a>notification</a>'s <a for=notification>require interaction preference flag</a>.
+<p>The <dfn attribute for=Notification><code>vibrate</code></dfn> getter steps are to return
+<a>this</a>'s <a>notification</a>'s <a for=notification>vibration pattern</a>.
 
-<p>The <dfn attribute for=Notification><code>data</code></dfn> attribute's getter must return
-<a abstract-op>StructuredDeserialize</a>(<a>notification</a>'s <a for=notification>data</a>,
-<a>this</a>'s <a>relevant Realm</a>). If this throws an exception, then return null.
+<p>The <dfn attribute for=Notification><code>timestamp</code></dfn> getter steps are to return
+<a>this</a>'s <a>notification</a>'s <a for=notification>timestamp</a>.
+
+<p>The <dfn attribute for=Notification><code>renotify</code></dfn> getter steps are to return
+<a>this</a>'s <a>notification</a>'s <a for=notification>renotify preference flag</a>.
+
+<p>The <dfn attribute for=Notification><code>silent</code></dfn> getter steps are to return
+<a>this</a>'s <a>notification</a>'s <a for=notification>silent preference flag</a>.
+
+<p>The <dfn attribute for=Notification><code>requireInteraction</code></dfn> getter steps are to
+return <a>this</a>'s <a>notification</a>'s
+<a for=notification>require interaction preference flag</a>.
```

Comment:
Probably these should be updated to be booleans? Right now it looks like you're returning a flag which isn't super well-defined (although of course any reader will know what you mean).